bin/webpack.js

```diff
@@ -441,10 +443,10 @@
 			}
 
 			if (outputOptions.infoVerbosity === "verbose") {
-				compiler.hooks.beforeCompile.tap("WebpackInfo", (compilation) => {
+				compiler.hooks.beforeCompile.tap("WebpackInfo", compilation => {
```

Comment:
I think maybe this is already commited in master

Reply:
addressed.